aspnetcore/tutorials/first-web-api.md

```diff
@@ -394,42 +394,41 @@ The following image shows the Postman update:
 
 ![Postman console showing 204 (No Content) response](first-web-api/_static/pmcput.png)
 
-### Delete
+## Add a Delete method
 
 Add the following `Delete` method:
 
 [!code-csharp[](first-web-api/samples/2.2/TodoApi/Controllers/TodoController.cs?name=snippet_Delete)]
 
 The `Delete` response is [204 (No Content)](https://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html).
 
-Use Postman to delete the to-do item:
+### Test the Delete method
+
+Use Postman to delete a to-do item:
 
 * Set the method to `DELETE`.
 * Set the URI of the object to delete, for example `https://localhost:5001/api/todo/1`
 * Select **Send**
 
-The sample app doesn't allow you to delete all the items. When there are no items, a new one is created.
+The sample app allows you to delete all the items, but when no item is left, a new one is created.
 
-## Call the web api with jQuery
+## Call the API with jQuery
 
 In this section, an HTML page is added that uses jQuery to call the web api. jQuery initiates the request and updates the page with the details from the API's response.
 
-Configure the app to serve static files and to enable default file mapping with the following calls:
-
-* [UseStaticFiles](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) 
-* [UseDefaultFiles](/dotnet/api/microsoft.aspnetcore.builder.defaultfilesextensions.usedefaultfiles#Microsoft_AspNetCore_Builder_DefaultFilesExtensions_UseDefaultFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) extension methods in *Startup.Configure*. For more information, see [Static files](xref:fundamentals/static-files).
+Configure the app to [serve static files](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_) and [enable default file mapping](/dotnet/api/microsoft.aspnetcore.builder.staticfileextensions.usestaticfiles#Microsoft_AspNetCore_Builder_StaticFileExtensions_UseStaticFiles_Microsoft_AspNetCore_Builder_IApplicationBuilder_):
 
 [!code-csharp[](first-web-api/samples/2.2/TodoApi/Startup.cs?highlight=14-15&name=snippet_configure)]
 
 ::: moniker range=">= aspnetcore-2.2"
-Create *wwwroot* directory to contain the assets in this section.
+Create a *wwwroot* folder in the project directory.
```

Comment:
The original didn't say where to create the wwwroot folder.  Adding "to contain..." made the revised sentence long without providing useful information, since you would only create a folder to hold something you're going to put into it anyway.

We could add a paragraph explaining the special function of the wwwroot folder in ASP.NET Core web apps, but that seems out of place in a tutorial about web API.